nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="nodes-secondary-scheduler-release-notes"]
+= {secondary-scheduler-operator-full} release notes
+include::_attributes/common-attributes.adoc[]
+:context: nodes-secondary-scheduler-release-notes
+
+toc::[]
+
+The {secondary-scheduler-operator-full} allows you to deploy a custom secondary scheduler in your {product-title} cluster.
+
+These release notes track the development of the {secondary-scheduler-operator-full}.
+
+For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
+
+[id="secondary-scheduler-operator-release-notes-1.0"]
+== Release notes for {secondary-scheduler-operator-full} 1.0
```

Comment:
@bergerhoffer should we say 1.0.0 instead of 1.0 as when searched on the operator hub that is what it shows.
1.0.0 provided by Red Hat, Inc.

Reply:
Yes definitely!

@ingvagabund Do you mind if I update the advisory too to use 1.0.0 instead of 1.0? Also, the title/description says "OpenShift Secondary Scheduler Operator Container", can I also update it to use the official name? So:

"Secondary Scheduler Operator for Red Hat OpenShift 1.0.0"

Reply:
Absolutely